hyperion-geiger-counter/src/main/java/com/willowtreeapps/hyperion/geigercounter/GeigerCounterModule.java

```diff
@@ -0,0 +1,67 @@
+package com.willowtreeapps.hyperion.geigercounter;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.willowtreeapps.hyperion.plugin.v1.PluginModule;
+
+class GeigerCounterModule extends PluginModule
+        implements View.OnClickListener {
+
+    private View view;
+    private DroppedFrameObserver observer;
+
+    // Helpers
+
+    private static boolean arePrerequisitesAvailable() {
+        return 16 <= Build.VERSION.SDK_INT;
```

Comment:
You could also use `Build.VERSION_CODES.ICE_CREAM_SANDWICH_MR1` rather than `16`

20x the keystrokes. Little bit more info.